src/app/legacy/psammead/psammead-paragraph/src/index.jsx

```diff
@@ -8,9 +8,8 @@ import { getSansRegular } from '#psammead/psammead-styles/src/font-styles';
 const Paragraph = styled.p`
   ${({ script }) => script && getBodyCopy(script)};
   ${({ service }) => getSansRegular(service)}
-  color: ${({ darkMode }) =>
-    props =>
-      darkMode ? props.theme.palette.LUNAR : props.theme.palette.GREY_10};
+  color: ${({ darkMode, theme }) =>
```

Comment:
I don't like the old `darkMode` and `theme.isDarkUi` mix, so will create a follow up task to remove `darkMode` and have that flag come from the ThemeProvider as well for the old OnDemandTV pages.

Reply:
I know that naming things is always the hardest thing to do! Could we use`isDarkMode` / `darkMode`, or is `isDarkUi` the preferred terminology?

Reply:
We want to avoid `DarkMode` as that can imply a setting that the user can turn on that we adjust styling too. Whereas this is a 'fixed' UI design choice.

Reply:
Ah yes, makes sense, thanks for the explanation. We might want to use `DarkMode` to mean something a user can configure in future 👍